day_7/10_speckled_frogs.rb

```diff
@@ -0,0 +1,29 @@
+x = 10
+plural = "frogs"
+singular = "frog"
+
+while x > 0
```

Comment:
Good job using iteration and string interpolation to create the speckled frogs rhyme.